src/main/java/gregtech/common/covers/detector/CoverDetectorEnergyAdvanced.java

```diff
@@ -25,25 +26,35 @@
 import net.minecraftforge.common.capabilities.Capability;
 
 import javax.annotation.Nonnull;
-import java.util.regex.Pattern;
+import java.util.ArrayList;
+import java.util.List;
 
 
 public class CoverDetectorEnergyAdvanced extends CoverBehavior implements CoverWithUI, ITickable, IControllable {
 
-    private static final Pattern ALLOWED_CHARS = Pattern.compile(".[0-9]*");
+    private static final int PADDING = 5, SIZE = 18;
 
-    public long minEU, maxEU;
+    private static final long DEFAULT_MIN_EU = 0, DEFAULT_MAX_EU = 2048;
+    private static final int DEFAULT_MIN_PERCENT = 33, DEFAULT_MAX_PERCENT = 66;
+
+    public long minValue, maxValue;
+    private long maxValueUpperLimit;
```

Comment:
This instance variable is unused.

Reply:
addressed with https://github.com/GregTechCEu/GregTech/pull/1511/commits/40ab59e1e7f54457e22a4a5e3cbb923a8e7dd0b5